docs/guides/publish.md

```diff
@@ -9,9 +9,14 @@ the PyPA tools [`build`](https://github.com/pypa/build) and
 Build your package with the official `build` frontend:
 
 ```console
-$ uvx --from build pyproject-build
+$ uvx --from build pyproject-build --installer uv
 ```
 
+!!! note
+
+    Using `--installer uv` is not required, but uses uv instead of the default, pip, for faster
+    builds.
```

Comment:
I would be fine with omitting this admonition.

Reply:
Me too, trying to explain the complex command more than anything.

Reply:
I'll leave it for now, the doc is so short.